nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-client-dto/src/main/java/org/apache/nifi/web/api/dto/CounterDTO.java

```diff
@@ -98,4 +98,12 @@ public void setValueCount(Long valueCount) {
         this.valueCount = valueCount;
     }
 
+    @Override
+    public CounterDTO clone() {
```

Comment:
@markap14 There appears to be about 5 instances of super.clone(). Given the ambiguous specification on the implementation of Object.clone and the inconsistency of doing a manual clone explicitly elsewhere can we update these?